src/pycram/external_interfaces/giskard.py

```diff
@@ -305,16 +304,12 @@ def achieve_joint_goal(goal_poses: Dict[str, float]) -> 'MoveResult':
     :param goal_poses: Dictionary with joint names and position goals
     :return: MoveResult message for this goal
     """
-    sync_worlds()
-    par_return = _manage_par_motion_goals(giskard_wrapper.set_joint_goal, goal_poses)
```

Comment:
This line has to stay here, it is used for the parallel execution of the language

Reply:
It's being called still in set_joint_goal which is called here